examples/tools/autotools/create_your_first_package.rst

```diff
@@ -0,0 +1,147 @@
+.. _examples_tools_autotools_create_first_package:
+
+Create your first Conan package with Autotools
+==============================================
+
+.. warning::
+
+  This example only works in Linux and OSX, but not in Windows (even if using msys2/cygwin subsystems,
+  WSL shouldn't be a problem it is Linux). There is some provision in Conan with ``win_bash = True``
+  to be able to work with Autotools in Windows subsystems, but not working in this example.
```

Comment:
Same suggestion than the other

```suggestion
.. warning::

  This example will only work for Linux and OSX environments and does not support Windows directly, including msys2/cygwin subsystems.
  However, Windows Subsystem for Linux (WSL) should work since it provides a Linux environment. While Conan offers `win_bash = True` 
  for some level of support in Windows environments with Autotools, it's not applicable in this tutorial.

```